lib/wallaby/dsl/finders.ex

```diff
@@ -13,6 +15,17 @@ defmodule Wallaby.DSL.Finders do
     do_find_elements(session, to_params(query))
   end
 
+  def wait_until_visible(session, query, opts \\ [])
+  def wait_until_visible(session, query, [{:count, count} | _] = _opts) do
+    retry fn ->
+      case all(session, query) do
+        elements when length(elements) == count -> elements
```

Comment:
I'm surprised that you can't do `.bench__availability:first`. You could definitely use xpath for it but I'd really like to avoid having that be a use case for people. I could see value in a `count: :any` but `:any` to me means that it could also be 0. Is that just me?